app/src/pages/explorer/Home.vue

```diff
@@ -78,6 +78,14 @@ export default {
   },
```

Comment:
I can't leave a comment on the actual line, but it's on line 69: `link: 'explorer/create'`

Reply:
fixed